src/styles.scss

```diff
@@ -1109,3 +1109,7 @@ hr {
 .mat-radio-button .mat-form-field-can-float.mat-form-field-should-float .mat-form-field-label {
   top: 1.8em;
 }
+
+.mat-checkbox.text-wrap > label.mat-checkbox-layout {
```

Comment:
https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist 

"Global styles that require the developer to actively use it..."